### DISCUSSION TITLE
Files from `/public` loose link on build

### DISCUSSION THREAD

**Discussion by educkf** [nuxt/nuxt#18114] [Questions] [answered]
I have some static logo images and stuff like that on `/public` folder, and while on dev, it is served correctly to the browser. I point to it like these on the vue files:

```
<img src="/ogo.png" >
```

But once I build using `NITRO_PRESET=firebase yarn build` and deploy to firebase, it looses the links to all images.

On served page to the browser, I see on source code it is linked like this:

```
<img src="/_nuxt/logo.png" >
```

...but if I go check on the  `.output/public` folder, image files are not on the `./nuxt` folder, but out of it, on the root of  `/public` folder. I have no special setup for aliases nor anything.

Just for testing, I moved all image files manually to inside the `./nuxt` file, redeployd the public folder, and then they are correctly shown on browser.

Is there any setup or special way to reference images I should be using? (if that is the case, I will probably make some sugestions to the respective pages on docs)

**Comment by viii47** [comment, marked as answer]
Hey @educkf,

this is a known issue - see #2261

**Reply by educkf** [reply]
Thanks! I will keep an eye on that one

**Reply by viii47** [reply]
@educkf you can bypass this issue temporarily when you drop your logos/static pictures into the asset folder - they'll build fine :)

**Reply by educkf** [reply]
In that case, should I point to the asset folder as follows?

```
<img src="/assets/logo.png" >
```

**Reply by viii47** [reply]
Just add `~` at the beginning and it should work, example from my current project:
```js
<template>
    <div>
        <NuxtLink :to="`/${$currentLocale()}`">
            <img src="~/assets/img/logo_prev.png" alt="logo" class="nav-logo" data-img="true" />
        </NuxtLink>
    </div>
</template>
```

**Reply by educkf** [reply]
Perfect, thanks!
